.gitignore

```diff
@@ -1,3 +1,5 @@
+*.iml
+.idea
```

Comment:
This should be in your user's gitignore, rather than the project's.

Reply:
Sure thing. I'll revert that.